error

[ERROR] addons/aarmory/lua/autorun/aarmory_init.lua:121: bad argument #1 to 'JSONToTable' (string expected, got no value)
  1. JSONToTable - [C]:-1
   2. fn - addons/aarmory/lua/autorun/aarmory_init.lua:121
    3. unknown - addons/ulib/lua/ulib/shared/hook.lua:109

--- a/aArmory/lua/autorun/aarmory_init.lua
+++ b/aArmory/lua/autorun/aarmory_init.lua
@@ -8,6 +8,7 @@ AddCSLuaFile("aarmory_localisation.lua")
 include("aarmory_config.lua")
 include("aarmory_localisation.lua")
 
+if (SERVER) then
 local function makeConfig()
     if !file.Exists("aarmory/config.txt", "DATA") then
         local configTable = {}
@@ -140,3 +141,4 @@ local function aarmorySpawn()
 
 end
 hook.Add( "InitPostEntity", "aarmorySpawn", aarmorySpawn )
+end

